convene-web/spec/furniture/check_dropbox/check_policy_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe Furniture::CheckDropbox::CheckPolicy, type: :policy do
+  let(:subject) { described_class } 
+  let(:space) { FactoryBot.build_stubbed(:space) }
+  let(:check) { instance_double(Furniture::CheckDropbox::Check, space: space) }
+  context "as a guest" do
+    let(:actor) { Guest.new }
+    permissions :show?, :update?, :edit?, :destroy? do
+      it { is_expected.not_to permit(actor, check) }
+    end
+
+    permissions :create?, :index? do
+      it { is_expected.to permit(actor, check) }
+    end
+  end
+
+  context 'as a neighbor' do
```

Comment:
You're using double quotes in the other `context` descriptions. :-)

Reply:
Doh! Thank you!